### ISSUE TITLE
Allow spaces between the notes in chords

### ISSUE THREAD

**Issue by newlandsvalley** [newlandsvalley/purescript-abc-parser#39]
i.e. [A B C] as well as [ABC].  I've been looking at ABC samples for 8 or 9 years now and have just come across my first example of this in the wild.

**Comment by newlandsvalley** [comment]
Fixed by 4d1344b12a7ce8fbecf9a8b9111fd911ab1cc00d in ```ps014``` branch.

**Comment by jgarte** [comment]
Should this be closed or not until it is merged into master?

**Comment by newlandsvalley** [comment]
Again, as soon as ```v2.0.0``` appears in the package-sets.

**Comment by newlandsvalley** [comment]
Merged into master although still not in package sets until the Registry import is fixed
